e2e/regression/guardrail.correction_range.spec.js

```diff
@@ -0,0 +1,50 @@
+const { LoopTest, screenName, target } = require('../../src/index');
+
+describe.skip('guard rail correction range', () => {
+    var loopTest;
+    it('should setup with correct configuration', async () => {
+        loopTest = await new LoopTest.Builder(target.tidepool)
+            .withSettings({ AddPumpSimulator: true })
+            .withStartScreen(screenName.settings)
+            .build();
+    });
+
+    // TODO: once CorrectionRanges uses accessibile pickers we should
+    // be able to launch loop once and then reuse the setup
+    it('maximum 180 can be set, warning', async () => {
+        await loopTest.settingsScreen.SetCorrectionRanges([{ time: '12:00 AM', min: '179', max: '180' }]);
```

Comment:
Any thoughts on how to represent these `mg/dL` values in `mmol/L` units for tests when (DIY|Tidepool) Loop are set to `mmol/L`? Consider putting these values in something separate that can be configured for `mg/dL` or `mmol/L` mode. Perhaps something like `limits.correctionRange.max.maximum`, `limits.correctionRange.max.warning`.